lib/travis/hub/model/build/matrix.rb

```diff
@@ -4,6 +4,10 @@ def finished?
       jobs.all?(&:finished?) || fast_finish? && required.finished?
     end
 
+    def created?
```

Comment:
I think it makes sense, since it will be only used on checking if the build can be restarted or not